CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- [Experimental Webview Provider](https://github.com/multiversx/mx-sdk-dapp/pull/1012)
```

Comment:
⚠️ Breaking change: Added Experimental Webview Provider

Reply:
This is not a breaking change. It's a new feature